Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/views/MoreInformationView.kt

```diff
@@ -14,9 +17,12 @@ class MoreInformationView @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
-) : ConstraintLayout(context, attrs, defStyleAttr) {
+) : ConstraintLayout(context, attrs, defStyleAttr), Checkable {
```

Comment:
Naming again :) Should me think about a different name now that the functionality of this view has increased? Maybe something like `SettingsRowView`?

Reply:
Chilja has created something that is more like the preference/settings views I think:
https://github.com/corona-warn-app/cwa-app-android/pull/3019

We can merge both first, as each improve existing code already, and then evaluate re-using/renaming. Let's see when we have time this sprint :cold_sweat: